src/pages/landing/index.test.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { http, HttpResponse, delay } from 'msw';
+import { setupServer } from 'msw/node';
+import { screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import { renderWithProviders } from 'utils/test-utils';
+import LandingPage from './';
+
+const server = setupServer(
+  http.get('/server-timestamp', async () => {
+    await delay(150);
+    return HttpResponse.json(1);
+  }),
+  http.post('/login', async () => {
+    await delay(150);
+    return HttpResponse.json('auth-token');
+  }),
+  http.get('/profile/current/subscriptions/active-features', async () => {
+    await delay(150);
+    return HttpResponse.json([]);
+  }),
+  http.get('/tiers', async () => {
+    await delay(150);
+    return HttpResponse.json([{
+      id: '1',
+      name: 'Tier 1',
+      subtitle: 'Tier subtitle',
+      features: [
+        { name: 'Feature 1', },
+        { name: 'Feature 2', }
+      ],
+      usdPrice: 100,
+      enabled: true,
+    }]);
+  }),
+  http.get('/ada-usd-price', async () => {
+    await delay(150);
+    return HttpResponse.json(10.0);
+  }),
+  http.put('/profile/current', async () => {
+    await delay(150);
+    return HttpResponse.json({});
+  }),
+  http.get('/profile/current/subscriptions', async () => {
+    await delay(150);
+    return HttpResponse.json([{ id: '1', status: 'pending', price: 10 }]);
+  }),
+  http.post('/profile/current/subscriptions/1', async () => {
+    await delay(150);
+    return HttpResponse.json({ id: '1' }, { status: 201 });
+  }),
+  http.get('/profile/current/balance', async () => {
+    await delay(150);
+    return HttpResponse.json(-1);
+  }),
+);
```

Comment:
Can we separate integration tests by some means, either into a different file with a different naming convention. As I think, we don't want to calculate the code coverage including them.

Reply:
I moved the integration test to another folder, but I don't know if we should exclude them from the code coverage calculation. Mostly because the integration tests are the ones that tests the reducers